test/unit/support.js

```diff
@@ -67,7 +67,27 @@ testIframeWithCallback( "Check CSP (https://developer.mozilla.org/en-US/docs/Sec
 			"pixelMarginRight": true,
 			"pixelPosition": true,
 			"radioValue": true,
-			"reliableMarginRight": true
+			"reliableMarginRight": true,
+			"reliableDimensionsInFullscreenMode": true
+		};
+	} else if ( /rv:11\.0/i.test( userAgent ) ) {
```

Comment:
IE 11 was already captured by the `trident\/7\.0` part in the regex below; they were grouped as all test results were the same, now they have to be divided.

I'd use `trident\/7\.0`, though and not `rv:11\.0` as that would conflict with Firefox 11. We don't test on Firefox 11 but it's always better to prevent confusion if possible. Also, looking at `trident` it's more clear we're dealing with IE than when looking at a generic `rv`.